sdk/storage/Azure.ResourceManager.Storage/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 1.0.0-beta.9 (Unreleased)
-
-### Features Added
+## 1.0.0-beta.9 (2022-05-13)
```

Comment:
I think we might miss two things here:

- Add the `RP` prefix for the private link related resources and models
- Add the update method using put if this resource doesn't have a patch method

Reply:
Thank you! Let me add them

Reply:
OK now in storage it is called StoragePrivateLinkResource, but it is not inheriting from ArmResource which violates our rules.
But considering this is a beta release, I will let it slide.